### ISSUE TITLE
Getting Started // Documentations is required, Unable to start the project

### ISSUE THREAD

**Issue by saurabhshahare** [Yuyz0112/syncit#14]
Hey thanks for syncit.

I wanted to try this out in my system, probably missing some steps to get this started. Can you please share the basic steps, requirements to follow?

**Comment by IridescentLiu** [comment]
it is 404 when i npm start in packages/ui

**Comment by saurabhshahare** [comment]
> 
> 
> it is 404 when i npm start in packages/ui

You need to access <URL>:5000/app.html
<URL>:5000/embed.html

I got the pages working but still I am not able to start the co-browsing somehow.

**Comment by IridescentLiu** [comment]
> > it is 404 when i npm start in packages/ui
> 
> You need to access :5000/app.html
> :5000/embed.html
> 
> I got the pages working but still I am not able to start the co-browsing somehow.

thanks but how can i register email? Did you login in the embed.html?

**Comment by saurabhshahare** [comment]
> 
> 
> > > it is 404 when i npm start in packages/ui
> > 
> > 
> > You need to access :5000/app.html
> > :5000/embed.html
> > I got the pages working but still I am not able to start the co-browsing somehow.
> 
> thanks but how can i register email? Did you login in the embed.html?

No need to register it's just a webpage to mimic the website. You can enable the Syncit co-browsing from widget on bottom right corner and put the same id on <>/app.html

**Comment by IridescentLiu** [comment]
> > > > it is 404 when i npm start in packages/ui
> > > 
> > > 
> > > You need to access :5000/app.html
> > > :5000/embed.html
> > > I got the pages working but still I am not able to start the co-browsing somehow.
> > 
> > 
> > thanks but how can i register email? Did you login in the embed.html?
> 
> No need to register it's just a webpage to mimic the website. You can enable the Syncit co-browsing from widget on bottom right corner and put the same id on <>/app.html

yes but i some examples in demo vedios, url is localhost:5000/examples/1-live.html.  Did u try to change the code in embed.html?

**Comment by scott-vector** [comment]
How does this work over the internet? I seem to be able to cobrowse on one machine (even if it is elsewhere on the internet) but for some reason when i try to use App on machine A and Embed on Machine B, it doesn't work.

**Comment by Yuyz0112** [comment]
@scott-vector You can try the `PeerjsTransporter ` or `PeerjsTransporter `

**Comment by scott-vector** [comment]
Ok. Is there a way we can get on chat about a few things? I have Telegram etc. I would also like to talk about a corporate sponsorship. I have the rrweb recording and replay working, I need help with the syncit. I also need to make sure I am optimizing both setups so I can prove some scalability.

**Comment by Yuyz0112** [comment]
@scott-vector Could you send me an email and I can invite you to the sponsor's slack channel.

**Comment by scott-vector** [comment]
Sure. ***@***.***

**Comment by scott-vector** [comment]
Just making sure you received. Would like to chat. Thanks.

**Comment by Yuyz0112** [comment]
@scott-vector I've not received it, could you send an email to aryu0112@gmail.com ?

**Comment by Juice10** [comment]
Hi @saurabhshahare, thanks for submitting this issue. The syncit project got [moved to the rrweb-io GitHub org](https://github.com/rrweb-io/syncit), and this repo will get archived soon. Would you mind re-submitting your issue there if it is still relevant for the [new 1.0.1 (rrweb 2.0 based) syncit](https://github.com/rrweb-io/syncit)?
